azure-pipelines.yml

```diff
@@ -12,48 +12,62 @@ jobs:
     vmImage: 'Ubuntu 16.04'
   strategy:
     matrix:
-      pip-gcc4:
-        gccVersion: '4.9'
+      pip-gcc8-py36:
```

Comment:
this is called gcc8, but below it says gccVersions=6

I'm not sure I understand why we want to add a new build

Reply:
We want to test both python 3.7 and 3.6 so that's two buildm then the conda ones with the examples and stuff

Reply:
Can have up to 10 at same time on pipeline so it' fine I think